### DISCUSSION TITLE
Feature request: file content indexation in nemo.

### DISCUSSION THREAD

**Discussion by PiotrVVV** [linuxmint/discussions#671] [Ideas]
Hi, 
Thank you for developing cinnamon! I have a feature idea:
please decide whenever you can enable file content indexation - text files, pdf, libreoffice, MS office documents, html files, mail. This feature is well-provided in KDE via baloo or gnome via tracker-miner. It makes working with large sets of documents much more efficient. Nemo has file content search feature, which in my case takes up to 20 minutes to search through files. It would be very convenient that nemo pulled tracker-miner indexed results and then perform search just like nautilus does things. If not by default, it could be implemented via en extension I suppose.
Best
P.

**Comment by tre-flip** [comment]
It would be great to have access to indexed search in the main menu of Cinnamon too. I think, nowadays many non-technical users expect this sort of functionality.

The downside it that indexing takes some system resources, so an option which disables it would be nice to have, in case if Cinnamon is running on low end hardware.
